docs/src/snap/howto/install/custom-bootstrap-config.md

```diff
@@ -47,19 +47,21 @@ The node will be 'Ready' to host workloads after the CNI is deployed successfull
 ## Bootstrap configuration file
 
 If your deployment requires a more fine-tuned configuration, use the bootstrap
-configuration file. By default the network, dns and gateway features
-are enabled, but the ingress, load-balancer and local-storage features are
-disabled.
+configuration file. 
 
+``` {note}
+By default all features including network, dns, gateway, ingress, load-balancer
```

Comment:
I would add when using a custom bootstrap configuration file. They are not set disabled by default if you use the generic bootstrap